contracts/evm/src/rollup/SFFLRegistryRollup.sol

```diff
@@ -145,6 +145,21 @@ contract SFFLRegistryRollup is Initializable, OwnableUpgradeable, Pausable, SFFL
         updateStateRoot(message, signatureInfo);
     }
 
+    /**
+     * @notice Forces an operator set update. This is meant to be used only
+     * by the owner in a testnet scenario in case there is no consensus on a
+     * particular operator set update. This can also be used while operator
+     * set updating is paused.
+     * @param message Operator set update message
+     */
+    function forceOperatorSetUpdate(OperatorSetUpdate.Message calldata message) external onlyOwner {
+        require(message.id == nextOperatorUpdateId, "Wrong message ID");
```

Comment:
There is no test that triggers this `require`

Reply:
Done in [61aaeaf](https://github.com/NethermindEth/near-sffl/pull/195/commits/61aaeafdae58cd6dd98631366990309177803f17)